wagtailio/project_styleguide/templates/patterns/base_page.html

```diff
@@ -135,6 +135,8 @@
         {% endif %}
 
         <script src="https://platform.twitter.com/widgets.js" async></script>
-        <script async defer src="https://buttons.github.io/buttons.js"></script>
+        {% block extra_js %}
+            <script async defer src="https://buttons.github.io/buttons.js"></script>
+        {% endblock %}
```

Comment:
Thought: I'm not sure we want to place this within the `{% block extra_js %}` as when this is used on other page templates, it will override the super (this one). As this is going to be in the header of every page I think it still makes sense for it to be where it was. What do you think?

Reply:
So I would say we don't want to add it to the base.html, we use that for the pattern-library and potentially in other non-page-based views like 404 and 500. A good place would be the header but that would suffer the same problem of being able to override it!

Reply:
The advice from https://buttons.github.io/ is to `<!-- Place this tag in your head or just before your close body tag. -->`. I personally prefer the latter as that's where I'd expect to find additional scripts that haven't been bundled but I guess the `defer` attribute won't hurt us if we place it in the header.

Reply:
> we use that for the pattern-library
Any reason why we wouldn't want to show this in the PL?

Reply:
Going to leave it here for the time being - we can always move it done the line if needs be.